docs/docs/clients.md

```diff
@@ -0,0 +1,246 @@
+# Client libraries
+
+Client libraries enable easy access to drand randomness in specific programming languages. They also provide piece of mind by _verifying_ randomness rounds and add additional features like failover, racing, aggregation, and caching.
+
+Drand randomness is available over HTTP, [gRPC](https://grpc.io/) and [libp2p PubSub](https://docs.libp2p.io/concepts/publish-subscribe/), but availability of transports other than HTTP in client libraries is implementation dependent.
+
+Choose a drand client library that matches the language in which your application is written:
+
+- [Go](#go)
+- [JS](#js)
+
+Language not listed? You can make raw requests to the [HTTP API](/docs/http-api/). If you have created a client library or know of a client library not listed here, [please let us know by opening an issue](https://github.com/drand/website/issues/new).
+
+## Go
+
+### Install
+
+```sh
+go get github.com/drand/drand
+```
+
+### Usage
```

Comment:
in the description of the base go client, we should provide explanation of the expected options that should be passed to `client.New`. These probably include:
* `WithCacheSize(128)`. (you probably want some caching in practice when using the client)
* `WithVerifiedResult(lastResult) / WithFullChainVerification()` - if you have persistent state and expect to be following the chain, you can get increased security by setting these.
* `WithAutoWatch()` - pre-load new results as they become available
* `WithPrometheus(r)` - report metrics on speed and performance to a provided prometheus registry.